GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+# Prometheus Governance 2.0
```

Comment:
High level comment: Goutham and myself decided to skip ~everything related to voting as this needs a wider discussion.